packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor.xml

```diff
@@ -426,12 +426,8 @@ Last updated: Jan 12 2024</description>
       </pruningSettings>
       <userId>1</userId>
     </metadata>
-    <dependentIds>
-      <string>cbd49d4a-24fa-40b6-aa12-d59d64168dd1</string>
-    </dependentIds>
-    <dependencyIds>
-      <string>8240f77a-19d4-4225-90fc-ed1c232286c3</string>
-    </dependencyIds>
+    <dependentIds/>
+    <dependencyIds/>
```

Comment:
Why removing dependency between channels?

Reply:
hmm this was done automatically

Reply:
Ill add back